src/sisp.test.js

```diff
@@ -59,12 +59,218 @@ describe('SISP Payment', () => {
       const referenceId = '1:3:testForm';
       const total = 89;
 
-      const sisp = new Sisp({posID, posAutCode});
+      const sisp = new Sisp({ posID, posAutCode });
       const result = await sisp.generatePaymentRequestForm(referenceId, total, webhookUrl);
 
       const expected = successData.replace(/\s/g, '');
 
       expect(result.replace(/\s/g, '')).toEqual(expected);
     });
   });
+
+  describe('Processing Payment Response', () => {
+    describe('Successful Payment', () => {
+
+      it('should return the correct response if messageType is 8', async () => {
```

Comment:
@FredSoares What should return the correct response?  I think that description can be more descriptive.

Reply:
@savylopes As you can see in the description of this issue, we just have one success case and the response will be always the same, and if you look at this test you will see the response will be the expected case `const expected = { code: 200, message: 'Payment processing success', data: paymentSuccessData }; `

Reply:
It is just for description if it fail the person that run the test can know exactly what return the wrong response if you improve it